lib/filter-and-transform/transforms/to-esri-attributes.js

```diff
@@ -0,0 +1,37 @@
+const { createIntegerHash } = require('../helpers')
+
+module.exports = function transformToEsriProperties (properties, geometry, dateFields, requiresObjectId, idField) {
+  requiresObjectId = requiresObjectId === 'true'
+  idField = idField === 'null' ? null : idField
```

Comment:
Is it case insensitive?

Reply:
It is, but `'null'` here comes from stringifying a `null` value internally so it will always be lowercase.

https://github.com/koopjs/winnow/blob/master/lib/normalize-query-options/id-field.js#L26

https://github.com/koopjs/winnow/blob/master/lib/sql-query-builder/select/fields-select-fragment.js#L18